src/mkdocs_include_markdown_plugin/event.py

```diff
@@ -616,12 +593,28 @@ def found_include_markdown_tag(  # noqa: PLR0912, PLR0915
 
         return text_to_include
 
-    markdown = tags['include'].sub(
-        found_include_tag,
+    get_file_content.ignore_paths = []
```

Comment:
This is crazy, please avoid setting arbitrary properties in the function itself.

Reply:
ok, if you don't like it, i could set the `ignore_paths` variable before `found_include_markdown_tag` and `found_include_tag,` function definitions. Or just add a new parameters to them, if you don't like that either.

Two variables for exclusions because i don't want to modify the ~global exclusions list with the local directives' exclusions. If in the same markdown we have two tags, one expecting to include (recursively) a file and the other directly excluding the same files, then—depending on the tag used—it would exclude the files for both tags anyway

Reply:
Looks fine. Note that mypy doesn't allow you to set an arbitrary property in the function definition.

Reply:
done!